docs/release-notes/generate-release-note.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+categories=("Bug Fixes" "Security Fixes" "Breaking Changes" "Improvements" "New Features")
+components=("WebUI" "Notebook" "TensorBoard" "Command" "Shell" "Experiment" "API" "Images")
+
+autocomplete() {
```

Comment:
This doesn't appear to actually allow for a write-in entry, and attempting to put one in causes "Invalid selection" and a newline to be included in the output. Looks like you want to examine `$REPLY` in the `else` branch. Also `autocomplete` is not a great name.